packages/client/test/rpc/eth/sendRawTransaction.spec.ts

```diff
@@ -31,6 +31,12 @@ describe(method, () => {
       return this
     }
     const common = new Common({ chain: Mainnet })
+    common
+      .hardforks()
+      .filter((hf) => hf.timestamp !== undefined)
+      .map((hf) => {
+        hf.timestamp = undefined
+      })
```

Comment:
I am not fully seeing through this change, but do we really need this hacky stuff here? 🤔 

Or is otherwise something wrong with our typing/data structure/whatever?